cio/io.go

```diff
@@ -255,3 +255,14 @@ func (l *logIO) Wait() {
 func (l *logIO) Close() error {
 	return nil
 }
+
+// Load the io for a container but do not attach
+//
+// Allows io to be loaded on task load as to deleted without
```

Comment:
minor nit: the "*as to deleted*" part of this comment may be missing a word/or needs a slight reword for clarity?